tutorials/tutorial1.ipynb

```diff
@@ -164,849 +174,1929 @@
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Step 4: Create genomic intervals to define regions for training and validation\n",
+    "The clean peak calls (`dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak`) were produced by MACS2 and are in .narrowPeak format. \n",
+    "Chromosome sizes files for the hg19 and hg38 human reference genomes are supplied with AtacWorks in the folder `AtacWorks/data/reference`. Here, we are using hg19.\n",
     "\n",
     "We need to define which regions of the genome will be used to train and test the model. We want to train models on some portion of the genome ('training set') and evaluate their performance on a separate portion ('validation set'). We will choose the model that performs best on the validation set as the best model. Later, we will evaluate the performance of this best model on a third portion of the genome ('holdout set').\n",
     "\n",
-    "We provide a chromosome sizes file 'hg19.auto.sizes' that contains sizes for all the autosomes of the hg19 reference genome. We split off chromosome 20 to use as the validation set, and chromosome 10 to use as the holdout set, and use the remaining autosomes as the training set. Since a whole chromosome is too long to feed into the model at once, we split each of these chromosomes into 50,000-bp long intervals."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-09 19:55:50,948:AtacWorks-intervals] Generating training intervals\n",
-      "INFO:2020-06-09 19:55:51,628:AtacWorks-intervals] Generating val intervals\n",
-      "INFO:2020-06-09 19:55:51,662:AtacWorks-intervals] Generating holdout intervals\n",
-      "INFO:2020-06-09 19:55:51,687:AtacWorks-intervals] Done!\n"
-     ]
-    }
-   ],
-   "source": [
-    "!python $atacworks/scripts/get_intervals.py \\\n",
-    "     --sizes $atacworks/data/reference/hg19.auto.sizes \\\n",
-    "     --intervalsize 50000 \\\n",
-    "     --out_dir ./ \\\n",
-    "     --val chr20 \\\n",
-    "     --holdout chr10"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This command generates three BED files in the current directory: `training_intervals.bed`, `val_intervals.bed`, and `holdout_intervals.bed`. These BED files contain 50,000-bp long intervals spanning the given chromosomes. We can look at these intervals:\n",
+    "We provide a chromosome sizes file 'hg19.auto.sizes' that contains sizes for all the autosomes of the hg19 reference genome. We split off chromosome 20 to use as the validation set, and chromosome 10 to use as the holdout set, and use the remaining autosomes as the training set. Since a whole chromosome is too long to feed into the model at once, we split each of these chromosomes into 50,000-bp long intervals.\n",
```

Comment:
I think instead of chromosome 20 we should use chromosome 2 as a validation set here

Reply:
done